ios/Classes/codec/AblyFlutterReader.m

```diff
@@ -146,26 +146,40 @@ use an explicitly received null from Dart (manifesting this side as a nil id)
 
     [o addAgent:@"ably-flutter" version: FLUTTER_PACKAGE_PLUGIN_VERSION];
 
-    AblyFlutterClientOptions *const co = [AblyFlutterClientOptions new];
-    ON_VALUE(^(const id value) {
-        [co initWithClientOptions: o hasAuthCallback: value];
-    }, dictionary, TxClientOptions_hasAuthCallback);
-
-    return co;
+    
+    return  [[AblyFlutterClientOptions alloc]
+             initWithClientOptions:o
+             hasAuthCallback:dictionary[TxClientOptions_hasAuthCallback]];
 };
```

Comment:
Are these changes supposed to be part of this PR 🤔

Reply:
Sorry @ikbalkaya, i had prematurely merged https://github.com/ably/ably-flutter/pull/252 into this PR by accident.